lib/testing/src/lib/core/pages/form/widgets/widget.ts

```diff
@@ -112,4 +113,8 @@ export class Widget {
     peopleWidget(): PeopleWidget {
         return new PeopleWidget();
     }
+
+    tab(): Tab {
```

Comment:
why do we need a function that creates new function? we can call "new Tab" instead